<launch>

 <arg name="mav_name"  default="MTD"/>
 <arg name="bag_name" default="flight"/>
 <arg name="bag_path" default="$(find rosplane)/../../../bags/2019-1-11"/>

 <rosparam command="load" file="$(find rosplane)/param/MTD.yaml" ns="controller"/>
 <node pkg="rosbag" type="play" name="player" output="screen" args="-q --clock $(arg bag_path)/$(arg bag_name).bag"/>

 <node name="estimator" pkg="rosplane" type="rosplane_estimator" output="screen">
   <param name="~use_inertial_sense" type="bool" value="true"/>
 </node>

 <!--Path Manager-->
 <node name="path_manager" pkg="rosplane" type="rosplane_path_manager" output="screen">
  <param name="R_min" value="50.0"/>
  <param name="loiter_radius" value="75.0"/>
  <param name="groundD" value="-6.7056"/>
  <param name="home_north" value="-100.0"/>
  <param name="home_east" value="-100.0"/>
  <param name="loiter_down" value="-150.0"/>
 </node>

 <!--Path Follower-->
 <node name="path_follower" pkg="rosplane" type="rosplane_path_follower" output="screen"/>

 <node name="controller" pkg="rosplane" type="rosplane_controller" output="screen">
   <param name="PWM_RAD_E" value="2.8"/>
   <param name="PWM_RAD_A" value="-1.6"/>
   <param name="PWM_RAD_R" value="1.8"/>
   <param name="ALT_TOZ" value="30.0"/>
   <param name="ALT_HZ" value="5.0"/>
   <param name="MAX_T" value="0.70"/>
   <param name="groundD" value="-6.7056"/>
 </node>

 <node name="recorder" pkg="rosbag" type="record" output="screen"
       args="-o flight_data_2019-1-11 /state /gps /controller_inners /controller_commands"/>

</launch>
